src/Main.java

```diff
@@ -1,9 +1,250 @@
+        } while (nextNumber != exitNumber);
 
 
+        //Zadatak 1c.
+
+        System.out.println("\nTask 1c: ");
+
+        Integer newTotal = null;
+        System.out.println("Please enter new Integer number: ");
+        int newIntNumber = scanner.nextInt();
+
+        if (newTotal == null) {
+            newTotal = 0;
+        } else {
+
+            newTotal += newIntNumber;
+        }
+
+        System.out.println("Total of " + newTotal + " and " + newIntNumber + " = " + newIntNumber);
+
+
+        //Zadatak 1d. and 1e.
+
+        System.out.println("\nTask 1d and 1e: ");
+
+        Integer newTotal1D = null;
+
+        System.out.println("\nPlease enter arithmetic operation (+, -, *, /): ");
+        String arithmeticOperator = scanner.next();
+
+        System.out.println("Please enter new Integer number: ");
+
+        int newIntNumber1D = scanner.nextInt();
+
+        if (newTotal1D == null) {
+            newTotal1D = 0;
+        }
+        switch (arithmeticOperator) {
+            case "+":
+                newTotal1D += newIntNumber1D;
+                break;
+            case "-":
+                newTotal1D -= newIntNumber1D;
+                break;
+            case "*":
+                newTotal1D *= newIntNumber1D;
+                break;
+            case "/":
+                if (newIntNumber1D == 0) {
+                    System.out.println("Can't divide by zero: ");
+                } else {
+                    newTotal1D /= newIntNumber1D;
+                }
+                break;
+        }
+
+        System.out.println("Total of " + newTotal1D + " and " + newIntNumber1D + " using " + arithmeticOperator + " = " + newTotal1D);
+
+        //Zadatak 2.
+
+        System.out.println("\nTask 2:");
+
+        int[] enteredNumbersArray = new int[100];
+        int enterNumber;
+        int count = 0;
+
+        do {
+            System.out.print("\nEnter any Integer number or (-10) to exit program: ");
+            enterNumber = scanner.nextInt();
+            if (enterNumber != -10) {
+                enteredNumbersArray[count] = enterNumber;
+                count++;
+            }
+        }while (enterNumber != -10);
+
+        int[] finalNumbersArray = new int[count];
+        for (int i = 0; i < count; i++) {
+            finalNumbersArray[i] = enteredNumbersArray[i];
+        }
+        System.out.println(Arrays.toString(finalNumbersArray));
+
+        //Zadatak 3.
+
+        System.out.println("\nTask 3: ");
+
+        int choice = 0;
+
+        while (choice != 4) {
+            System.out.println("\nCHOOSE OPTION FROM MENU: ");
+            System.out.println("1. Is string a palindrome");
+            System.out.println("2. Filter an Array");
+            System.out.println("3. Is number divisible by two");
+            System.out.println("4. Exit");
+
+            choice = scanner.nextInt();
+            switch (choice) {
+                case 1:
+                    System.out.print("Enter a String: ");
```

Comment:
why not extract everything specific to palindrome to its own method?
same applies for other cases